cinder_service_check.py

```diff
@@ -53,6 +53,12 @@ def check(auth_ref, args):
 
     services = r.json()['services']
 
+    # We need to match against a host of X and X@lvm (or whatever backend)
+    if args.host:
+        backend = ''.join((args.host, '@'))
```

Comment:
Actually I didn't review this otherwise I would have brought it up in the original review.